pkgs/os-specific/linux/util-linux/default.nix

```diff
@@ -106,6 +111,18 @@ stdenv.mkDerivation rec {
   enableParallelBuilding = true;
 
   postInstall = ''
+    moveToOutput bin/mount "$mount"
+    moveToOutput bin/umount "$mount"
+    ln -svf "$mount/bin/"* $bin/bin/
```

Comment:
I'm generally fearful of bash globs. I know this works as expected, but nonetheless is this fear too paranoid in this case?

Reply:
I think so; the one weird case here would be if nothing exists in $mount/bin (so it expands to nothing rather than the literal glob due to stdenv's `nullglob`) and we end up with no argument, and ln creates a symlink to $bin/bin in the working directory.